example/pythonlib/basic/1-simple/qux/src/main.py

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/python3
 import numpy as np
```

Comment:
I left this comment on the other PR as well, but let's swap out `numpy` for `jinja2` as our example third-party library. It is a lot more light weight and better mirrors what the other examples are doing. We should have some scientific/ml python stack examples somewhere but maybe not in the first example

Reply:
@lihaoyi Actually, i am carrying this after merging to finish line by adding examples and there i will add all the examples accordingly the reviews you made into my PR